fixing wrong array key type for PHP 8.0

TheCodingHood opened the pull request:
Fixing [TypeError] str_replace(): Argument #2 ($replace) must be of type array|string, int given 
on PHP 8

MarkusFoitzikC24 committed:
fixing wrong array key type for PHP 8.0

fe3dback commented:
can you please fix travis for this project? i have no time for this project any more, but green ci status required to merge pr.

it seems composer updated in travis to version 2, and current lock is broken with latest php/composer version.

MarkusFoitzikC24 committed:
upgrade composer dependencies

MarkusFoitzikC24 committed:
change phpunit version

MarkusFoitzikC24 committed:
Fixed Unittest in PHP 7.2 and set min php-version to 7.2

TheCodingHood commented:
> can you please fix travis for this project? i have no time for this project any more, but green ci status required to merge pr.
> 
> it seems composer updated in travis to version 2, and current lock is broken with latest php/composer version.

I have updated all dependencies and adapted the unit tests. Unfortunately, the dev dependencies now require a minimum of PHP 7.2. I have also adjusted this.

fe3dback commented:
thanks, it's great job
